### PR TITLE
completed issue-2-view entries

### PR DESCRIPTION
**Pull request by Rachael-Kim**
issue2

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -2,7 +2,22 @@
...
+      nextEntryID: 1
+    },
+    {
+      title: 'JavaScript',
+      image: './images/placeholder-image-square.jpg',
+      notes: `JavaScript, often abbreviated as JS, is a programming language that conforms to the ECMAScript
+              specification.`,
+      nextEntryID: 2
+    }
+  ],
```
Comment (scbowler) [inline js/data.js]: Here you shouldn't have any hard coded entries, entries should start as an empty array.

File: js/data.js [hunk 2]
```diff
@@ -2,7 +2,22 @@
...
   editing: null,
   nextEntryId: 1
 };
```
Comment (scbowler) [inline js/data.js:8]: In the `data.js` file you should be reading the data from `localStorage` and have an event listener for 'beforeunload` to save the data back to local storage when the page is closed. Refer to the `javascript-local-storage` exercise for reference.

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,87 @@
...
+    title: titleInput.value,
+    image: imgInput.value,
+    notes: notes.value,
+    nextEntryId: data.nextEntryId
+  };
+  data.entries.unshift(entry);
+  data.nextEntryId++;
+  imgPlaceholder.setAttribute('src', './images/placeholder-image-square.jpg');
+  formEntry.reset();
+  localStorage.setItem('data', JSON.stringify(data));
```
Comment (scbowler) [inline js/main.js]: Once your `data.js` file is properly setup you wont need to directly access `localStorage` in this file anywhere

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just needs some cleanup of how `localStorage` is being used. Let us know if you have any questions.

**Comment by Rachael-Kim** [comment]
issue2 revised

**Comment by Rachael-Kim** [comment]
issue2 revised1

**Comment by Rachael-Kim** [comment]
issue2 revised2

**Pull request merged**